scripts/scil_compute_pca.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to compute PCA analysis on diffusion metrics. Output returned is all significant principal components
+(e.g. presenting eigenvalues > 1) in a connectivity matrix format. This script can take into account all
+edges from every subject in a population or only non-zero edges across all subjects.
+
+The script can take directly as input a connectoflow output folder. Simply use the --connectoflow flag. For
+other type of folder input, the script expects a single folder containing all matrices for all subjects. Example:
+                                                            input_folder/sub-01_ad.npy
+                                                                        /sub-01_md.npy
+                                                                        /sub-02_ad.npy
+                                                                        /sub-02_md.npy
+                                                                        /...
+
+Output connectivity matrix will be saved next to the other metrics in the input folder. The plots and tables
+will be outputted in the designated folder from the <output> argument.
```

Comment:
Could you explain a bit how to start interpretation? I ran your test data  with the command:
scil_compute_pca.py ./ ../lol --list_ids test.text --metrics ad rd fa md --connectoflow

and my first column is : PC1	0.516503256371792	0.496327739954754	0.473191960436531	0.512808472323706 (in one sentence how do you interpret PCA, you can use chatGPT or stack overflow, but the user needs a starts or a least a link to a good resource for beginner (simpler than the paper, what is PCA)

Also, can you fix this kind of display overlap <errors>?
![image](https://user-images.githubusercontent.com/10820351/223445666-1459ff60-44ff-44a0-a42d-1e7a246cdb10.png)
![image](https://user-images.githubusercontent.com/10820351/223445749-5be2691d-c598-41db-9fdf-45a9815bf03a.png)